lib/stimulus_reflex/reflex.rb

```diff
@@ -133,9 +133,13 @@ def params
   def dom_id(record, prefix = nil)
     return "#" + [record.model_name.plural, prefix].compact.join("_") if record.is_a?(ActiveRecord::Relation)
```

Comment:
@leastbad you probably want to reverse the order also on L134

Reply:
🤦